test-suite/src/test/scala/org/scalajs/testsuite/jsinterop/TupleTest.scala

```diff
@@ -0,0 +1,42 @@
+/*                     __                                               *\
+**     ________ ___   / /  ___      __ ____  Scala.js Test Suite        **
+**    / __/ __// _ | / /  / _ | __ / // __/  (c) 2013, LAMP/EPFL        **
+**  __\ \/ /__/ __ |/ /__/ __ |/_// /_\ \    http://scala-js.org/       **
+** /____/\___/_/ |_/____/_/ | |__/ /____/                               **
+**                          |/____/                                     **
+\*                                                                      */
+package org.scalajs.testsuite.jsinterop
+
+import org.scalajs.jasminetest.JasmineTest
+
+import scala.scalajs.js
+
+object TupleTest extends JasmineTest {
+
+  describe("scala.scalajs.js.Tuple") {
+
+    it("should provide an equivalent of scala tuple") {
+      val obj: js.Tuple2[Int, String] = (42, "foobar")
+
+      expect(obj._1).toEqual(42)
+      expect(obj._2).toEqual("foobar")
+    }
+
+    it("should convert to scala tuple") {
+      val obj: js.Tuple2[Int, String] = (42, "foobar")
```

Comment:
Consider
```scala
val obj = js.Tuple2(42, "foobar")
```
as a shorter form to directly create a JS tuple without going through the Scala tuple. It also makes this test independent from the conversion from Scala tuple to JS tuple.